src/AppInstallerCLIInterfaceServer/AppInstallerCLIInterfaceServer.vcxproj

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Per compliance, for product code, we need to enable sdl check, code analysis and binskim. You can see how to enable them in this pr: [https://github.com/microsoft/winget-cli/pull/856](https://github.com/microsoft/winget-cli/pull/856)